src/Tag/Select.php

```diff
@@ -90,6 +90,19 @@ public function items(Optgroup|Option ...$items): self
         return $new;
     }
 
+    /**
+     * @see https://html.spec.whatwg.org/multipage/webappapis.html#handler-onchange
+     * @see https://html.spec.whatwg.org/multipage/indices.html#event-change
+     *
+     * @param string $value The script to be run when select value is changed.
+     */
+    public function onChange(string $value): self
```

Comment:
I don't think we should add it:

1. It's not a good practice to use such attributes with inline JavaScript.
2. There are tons of such events for every tag. Are we going to add every possible event?

Reply:
1. I agree, this HTML attribute is rare used. However, I have needed it a few times, and make it with this method more convient than `attribute()` method usage.

2. Step by step, when they are needed in practice.

Reply:
I don't think it is a good idea:

![image](https://github.com/user-attachments/assets/7ac9f9ee-4814-4738-8a5e-1816435afd67)

Reply:
👌